README.md

```diff
@@ -334,6 +334,48 @@ Remote backends like S3 is also supported. When a remote backend is used in your
 
 Just specify the path to that file, so that `vals` is able to transparently make the remote state contents available for you.
 
+### Terraform in GCS bucket (tfstategs)
+
+- `ref+tfstategs://bucket/path/to/some.tfstate/RESOURCE_NAME`
+
+Examples:
+
+- `ref+tfstate://bucket/path/to/some.tfstate/google_compute_disk.instance.id`
+
+It allows to use Terraform state stored in GCS bucket with the direct URL to it. You can try to read the state with command: 
+
+```
+$ tfstate-lookup -s gs://bucket-with-terraform-state/terraform.tfstate google_compute_disk.instance.source_image_id
+5449927740744213880
+```
+
+which is equivalent to the following input for `vals`:
+
+```
+$ echo 'foo: ref+tfstategs://bucket-with-terraform-state/terraform.tfstate/google_compute_disk.instance.source_image_id' | vals eval -f -
+```
+
+### Terraform in S3 bucket (tfstates3)
+
+- `ref+tfstates3://bucket/path/to/some.tfstate/RESOURCE_NAME`
+
+Examples:
+
+- `ref+tfstate://bucket/path/to/some.tfstate/aws_vpc.main.id`
```

Comment:
```suggestion
- `ref+tfstates3://bucket/path/to/some.tfstate/aws_vpc.main.id`
```

This should be `tfstates3` as well, right?

Reply:
Yeah of course